cmd/server/app/root.go

```diff
@@ -55,8 +55,8 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&logType, "log_type", "dev", "logger type to use (dev/prod)")
 
 	rootCmd.PersistentFlags().String("gcp_private_ca_parent", "", "private ca parent: /projects/<project>/locations/<location>/<name>")
-	rootCmd.PersistentFlags().String("oidc-issuer", "https://accounts.google.com", "OIDC provider to be used to issue ID token")
-	rootCmd.PersistentFlags().String("oidc-client-id", "237800849078-rmntmr1b2tcu20kpid66q5dbh1vdt7aj.apps.googleusercontent.com", "client ID for application")
+	rootCmd.PersistentFlags().String("oidc-issuer", "https://oauth2.sigstore.dev", "OIDC provider to be used to issue ID token")
```

Comment:
should be `https://oauth2.sigstore.dev/auth`

Reply:
done